ARMeilleure/CodeGen/RegisterAllocators/HybridAllocator.cs

```diff
@@ -86,6 +86,11 @@ public AllocationResult RunPass(ControlFlowGraph cfg, StackAllocator stackAlloc,
             var localInfo = new LocalInfo[cfg.Blocks.Count * 3];
             int localInfoCount = 0;
 
+            void ThrowNotVisisted()
```

Comment:
Same here on these two.
ThrowNotVisisted -> ThrowNotVisited
```suggestion
            void ThrowNotVisited()
```